.circleci/config.yml

```diff
@@ -127,6 +127,7 @@ jobs:
       - persist_to_workspace:
           root: /home/circleci/
           paths:
+            - oppia/core/domain/proto/
```

Comment:
Is this the location where the generated protos live, or just the proto definitions? If the former, I'd expect the generated files to go to a temporary generated folder.

Reply:
Hi, yes, this is location where generated proto files live. However, we can't generate files ones the gae app is deployed and hence these files have to be part of the deployment package. That's why they are in sort of permanent location. This location is same on Oppia-ml as well.

Reply:
Can we add a gitignore to make sure these don’t get uploaded or is that done already?

Reply:
I see that it is ignored--in that case, I think this is probably reasonable. Thanks!